src/Agent.Worker/Release/ReleaseTrackingManager.cs

```diff
@@ -171,11 +172,14 @@ private void PrintOutDiskUsage(IExecutionContext context)
                 var workDirectoryDrive = new DriveInfo(HostContext.GetDirectory(WellKnownDirectory.Work));
                 long freeSpace = workDirectoryDrive.AvailableFreeSpace;
                 long totalSpace = workDirectoryDrive.TotalSize;
-#if OS_WINDOWS
-                context.Output($"Working directory belongs to drive: '{workDirectoryDrive.Name}'");
-#else
-                context.Output($"Information about file system on which working directory resides.");
-#endif
+                if (PlatformUtil.RunningOnWindows)
```

Comment:
I feel like this block of code is repeated

Reply:
It is. I didn't try to clean that up.